source/_integrations/modbus.markdown

```diff
@@ -768,9 +735,9 @@ climates:
         word_byte:
           description: "Swap word ABCD -> DCBA, **not valid with data types: `int16`, `uint16`**"
     target_temp_register:
-      description: "Register address for target temperature (Setpoint) and default one also in case of hvac_mode_target_temp_reg_values usage."
+      description: "Register address for target temperature (Setpoint). Using a list, multiple registers can be defined for each of the available HVAC Mode. The list has to have a fixed size of 7 registers like the 7 available HVAC Modes, having the following relationship: Register **1: HVAC AUTO mode**; Register **2: HVAC Cool mode**; Register **3: HVAC Dry mode**; Register **4: HVAC Fan only mode**; Register **5: HVAC Heat mode**; Register **6: HVAC Heat Cool mode**; Register **7: HVAC OFF mode**. It is possible to set duplicated values for the modes where the devices has not a related register." 
```

Comment:
This does not sound correct....can you really define multiple registers for each HVAC mode ?

Which device needs multiple registers to set a single HVAC mode ?

Reply:
Ehm.. not really, I forgot to add "one" word. I'll do it. Thanks. 
It should be: `Using a list, multiple registers can be defined **one** for each of the available HVAC Mode`

Reply:
Well the new text is not a lot more understandable.....you can define one register pr HVAC mode, no need to talk about multiple registers.